### DISCUSSION TITLE
Skip current countdown

### DISCUSSION THREAD

**Discussion by Pramuspl** [YetAnotherPomodoroApp/YAPA-2#88] [Ideas]
Would it be possible to add a feature to step over the current countdown like here https://pomofocus.io/ ?

**Comment by floatas** [comment]
I'm not sure where should we place that button 🤔 when we start pomodoro on the left side we have {minimize, close}, {pause}, {stop} and {pomodoro index}, it's already full height of the timer.

**Reply by Pramuspl** [reply]
Maybe it's worth considering to use Play/Stop/Forward icons instead of text? They are pretty much self explanatory and don't take up as much space as text

**Reply by floatas** [reply]
It would be nice to have a mockup, how it would look like. If we add these small buttons then we probably will have two buttons per line and it might look clustered and confusing
